charts/talos-cloud-controller-manager/templates/deployment.yaml

```diff
@@ -65,6 +65,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
+            - name: KUBERNETES_SERVICE_HOST
```

Comment:
isn't it easier to use hostnetwork?

Reply:
KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT are using for go-kubernetes client connection.

Node can have IPv4 or IPv6 and `status.podIP` will have preferred IP to reach the node. 
I am not sure about 127.0.0.1 or ::1 addresses.

What do you think?

Reply:
If it's better I'm all for it :thumbsup